onnx/cpp2py_export.cc

```diff
@@ -58,7 +58,8 @@ std::unordered_map<std::string, py::bytes> CallNodeInferenceFunction(
     const py::bytes& nodeBytes,
     std::unordered_map<std::string, py::bytes> valueTypesByNameBytes,
     std::unordered_map<std::string, py::bytes> inputDataByNameBytes,
-    std::unordered_map<std::string, py::bytes> inputSparseDataByNameBytes) {
+    std::unordered_map<std::string, py::bytes> inputSparseDataByNameBytes,
+    std::unordered_map<std::string, py::bytes> outerScopeValueTypesByNameBytes) {
```

Comment:
Do we need this extra-parameter? I think the parameter `valueTypesByName` can serve both purposes. I realize that the underlying APIs distinguish them, but it seems simpler to manage with a single map for the python API, unless there is a good reason to make the distinction. I think there is no need since the same set of names are visible and can be used as node-inputs or inside graph-attributes.

Reply:
Thanks for pointing out! I was also confused here, because I thought there might be some use cases for different maps. I have updated it with single map. PTAL.